getting-started/running-the-libraries.md

```diff
@@ -0,0 +1,12 @@
+# Running the libraries
```

Comment:
I think this should be renamed to: How to run the Valtimo project locally

On a side note, does it really make sense for this documentation to be here? If anything, I think this needs to go into the project repositories themselves (e.g. in their respective READMEs) rather than here in the documentation.